src/app/components/MostRead/Amp/getRemoteDataScript/index.ts

```diff
@@ -0,0 +1,44 @@
+import { Services } from '../../../../models/types/global';
+import { serviceNumerals } from '../../Canonical/Rank';
+
+export const transformData = () => {
+  return `
+    data.items = data.records || [];
```

Comment:
This is temporary and will go away once the BFF Most Read response has been simplified